### ISSUE TITLE
Feature Request: fa-align-middle class

### ISSUE THREAD

**Issue by Miserlou** [FortAwesome/Font-Awesome#2797]
By default, fa-2x and higher icons look kind of off when used in some places (buttons, badges, other non-header places) because they have vertical-align: bottom;

It'd be nice if we could just add 'fa-align-middle' to an icon to have it be vertical-align: middle'd.

Other people are discussing this, so it could be a useful addition: http://stackoverflow.com/questions/17309928/how-to-center-text-vertically-with-a-large-font-awesome-icon

**Comment by ADTC** [comment]
Agreed. Any update on this? I'm just about to create my own custom class for it:

    .fa-middle {
        vertical-align: middle !important;
    }

At least there should be classes for `top`, `middle` and `bottom` from the [various choices](http://www.w3schools.com/cssref/pr_pos_vertical-align.asp).

PS: `fa-align-middle` can be confusing as there's already an icon called `fa-align-center`

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
